posturebest/Views/Torso3DView.swift

```diff
@@ -57,24 +49,25 @@ struct Model3DView: UIViewRepresentable {
         }
         
         //Fix from upside down imported position
-        skeletonNode.eulerAngles.x = .pi
-        
-        // Calculate the bounding box and center of the skeletonNode
-        let (min, max) = skeletonNode.boundingBox
-        let center = SCNVector3(
-            (min.x + max.x) / 2,
-            (min.y + max.y) / 2,
-            (min.z + max.z) / 2
-        )
-        print("Center of Skeleton: \(center)")
-
-        skeletonNode.position = SCNVector3(-center.x, -center.y - 1, -center.z)
+        skeletonNode.eulerAngles.y = .pi
 
         scene.rootNode.addChildNode(skeletonNode)
         context.coordinator.modelNode = skeletonNode
-
-        // Print the node hierarchy for debugging
-        printNodeHierarchy(node: scene.rootNode)
+        
+        modelHelper.setRootNode(node: skeletonNode)
+        
+        // set nodes for live updating
+        if let Human = scene.rootNode.childNode(withName: "HumanMale", recursively: true) {
+            let lowerBackNode = Human.skinner?.skeleton?.childNode(withName: "LowerBack", recursively: true)!
+            let midBackNode = Human.skinner?.skeleton?.childNode(withName: "MidBack", recursively: true)!
+            let upperBackNode = (Human.skinner?.skeleton?.childNode(withName: "UpperBack", recursively: true))!
+                        
+            modelHelper.setReferenceOrientation(orientation: lowerBackNode!.simdWorldOrientation)
+            modelHelper.setMidBackNode(node: midBackNode!)
+            modelHelper.setUpperBackNode(node: upperBackNode)
```

Comment:
why exclamation on lower and middle nodes but not here? not necessarily bad, just curious